### ISSUE TITLE
README installation instructions fail

### ISSUE THREAD

**Issue by ohad** [stefan-hoeck/idris2-pack#2]
# To reproduce
```
$ idris2 --version
Idris 2, version 0.5.1-36837942d
$ build/exec/pack --switch unstable-220429
Cloning project
Cloning into '/home/ohad/.pack/tmp'...
remote: Enumerating objects: 40294, done.
remote: Total 40294 (delta 0), reused 0 (delta 0), pack-reused 40294
Receiving objects: 100% (40294/40294), 27.29 MiB | 4.76 MiB/s, done.
Resolving deltas: 100% (27605/27605), done.
Changing into repo directory
Checking out to 4256cd15fdf1331ec6ab3140f73676100dedf4ae
Note: switching to '4256cd15fdf1331ec6ab3140f73676100dedf4ae'.

You are in 'detached HEAD' state. You can look around, make experimental
[.. lots of output ..]
Install API
echo "-- @""generated" > src/IdrisPaths.idr
echo 'module IdrisPaths' >> src/IdrisPaths.idr
echo 'export idrisVersion : ((Nat,Nat,Nat), String); idrisVersion = ((0,5,1), "4256cd15f")' >> src/IdrisPaths.idr
echo 'export yprefix : String; yprefix="/home/ohad/.pack/unstable-220429"' >> src/IdrisPaths.idr
idris2 --install idris2api.ipkg
  1/254: Building Libraries.Data.Fin (src/Libraries/Data/Fin.idr)
Error: Error in TTC
file: TTC data is in an newer format, file: /home/ohad/.pack/unstable-220429/idris2-0.5.1/prelude-0.5.1/Prelude.ttc, expected version: 2022041300, actual version: 2022042500 (the most likely case is that the ./build directory in your current project contains files from a previous build of idris2 or the idris2 executable is from a different build than the installed .ttc files)
make: *** [Makefile:180: install-api] Error 1
Error when executing system command.
Command: make install-api IDRIS2_PREFIX="/home/ohad/.pack/unstable-220429"
Error code: 2
```

**Comment by stefan-hoeck** [comment]
Thanks. Will look into this.

**Comment by ohad** [comment]
Thanks! It now dies at a different spot:

```
$ build/exec/pack switch unstable-220429
Cloning into '/home/ohad/.pack/tmp'...
remote: Enumerating objects: 40294, done.
remote: Total 40294 (delta 0), reused 0 (delta 0), pack-reused 40294
Receiving objects: 100% (40294/40294), 27.98 MiB | 5.76 MiB/s, done.
Resolving deltas: 100% (27560/27560), done.
Note: switching to '4256cd15fdf1331ec6ab3140f73676100dedf4ae'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 4256cd15 [ highlighting ] case trees in `:di` (#2440)
make[1]: Entering directory '/home/ohad/.pack/tmp/support/c'
cc -Wall  -fPIC -O2   -c -o idris_file.o idris_file.c
cc -Wall  -fPIC -O2   -c -o idris_util.o idris_util.c
cc -Wall  -fPIC -O2   -c -o idris_net.o idris_net.c
cc -Wall  -fPIC -O2   -c -o getline.o getline.c
cc -Wall  -fPIC -O2   -c -o idris_term.o idris_term.c
cc -Wall  -fPIC -O2   -c -o idris_system.o idris_system.c
cc -Wall  -fPIC -O2   -c -o idris_directory.o idris_directory.c
cc -Wall  -fPIC -O2   -c -o idris_memory.o idris_memory.c
cc -Wall  -fPIC -O2   -c -o idris_signal.o idris_signal.c
cc -Wall  -fPIC -O2   -c -o idris_support.o idris_support.c
ar rc libidris2_support.a idris_file.o idris_util.o idris_net.o getline.o idris_term.o idris_system.o idris_directory.o idris_memory.o idris_signal.o idris_support.o
ranlib libidris2_support.a
cc -shared -o libidris2_support.so idris_file.o idris_util.o idris_net.o getline.o idris_term.o idris_system.o idris_directory.o idris_memory.o idris_signal.o idris_support.o 
make[1]: Leaving directory '/home/ohad/.pack/tmp/support/c'
make[1]: Entering directory '/home/ohad/.pack/tmp/support/refc'
cc -Wall  -fPIC -O2   -c -o casts.o casts.c
cc -Wall  -fPIC -O2   -c -o prim.o prim.c
cc -Wall  -fPIC -O2   -c -o runtime.o runtime.c
cc -Wall  -fPIC -O2   -c -o clock.o clock.c
cc -Wall  -fPIC -O2   -c -o refc_util.o refc_util.c
cc -Wall  -fPIC -O2   -c -o conCaseHelper.o conCaseHelper.c
cc -Wall  -fPIC -O2   -c -o threads.o threads.c
cc -Wall  -fPIC -O2   -c -o stringOps.o stringOps.c
cc -Wall  -fPIC -O2   -c -o buffer.o buffer.c
cc -Wall  -fPIC -O2   -c -o mathFunctions.o mathFunctions.c
cc -Wall  -fPIC -O2   -c -o memoryManagement.o memoryManagement.c
ar rc libidris2_refc.a casts.o prim.o runtime.o clock.o refc_util.o conCaseHelper.o threads.o stringOps.o buffer.o mathFunctions.o memoryManagement.o
ranlib libidris2_refc.a
make[1]: Leaving directory '/home/ohad/.pack/tmp/support/refc'
make[1]: Entering directory '/home/ohad/.pack/tmp/support/chez'
# start library header
echo "(library (support) (export" > support-sep.ss
# print the list of exports
cat support.ss \
	| sed -n 's|(define (\?\([^ )]*\).*|\1|p' \
	>> support-sep.ss
echo ") (import (chezscheme))" >> support-sep.ss
# copy the code
cat support.ss >> support-sep.ss
# close the bracket
echo ") ; end of (library)" >> support-sep.ss
make[1]: Leaving directory '/home/ohad/.pack/tmp/support/chez'
/bin/sh: 1: --quiet: not found
mkdir -p bootstrap-build/idris2_app
cp support/c/libidris2_support.so bootstrap-build/idris2_app/
sed 's/libidris2_support.so/libidris2_support.so/g; s|__PREFIX__|/home/ohad/.pack/tmp/bootstrap-build|g' \
	bootstrap/idris2_app/idris2.ss \
	> bootstrap-build/idris2_app/idris2-boot.ss
/bin/sh ./bootstrap-stage1-chez.sh
Required SCHEME or IDRIS2_VERSION env is not set.
Invoke with SCHEME=[name of chez scheme executable]
make: *** [Makefile:249: bootstrap] Error 1
Error when executing system command.
Command: make bootstrap PREFIX="/home/ohad/.pack/unstable-220429"
Error code: 2
```
I have chezscheme installed (under `scheme` or `chezscheme9.5`). I remember this was an issue with the idris installer and @gallais (or others?) came up with some scripts to find the right executable.

**Comment by stefan-hoeck** [comment]
Ah yes. I knew I'd need a `--scheme` command line argument for people who have not set the `$SCHEME` variable. Will come up with a PR, after I implemented support for deleting installed packages (almost done).

Thanks, ohad, for trying this out and for your patience.

**Comment by stefan-hoeck** [comment]
Just merged another PR, adding a `--scheme` command line arg (see updated installation instructions). Feel free to give this another go. Make sure to run `build/exec/pack update-db` before running the `switch` command to make sure (if all goes well) that the latest version of `pack` is installed. I think I'll need a `pack update` command next.

**Comment by ohad** [comment]
Seems to work for me now!

**Comment by ohad** [comment]
Fixed by #6
